articles/databox/data-box-disk-deploy-picked-up.md

```diff
@@ -152,8 +152,24 @@ If needed, you can contact Quantium Solution Support (Japanese language) at the
 
    If you encounter any issues, kindly reach out to Data Box Operations Asia at adbo@microsoft.com. Provide the job name in the subject header and the issue encountered.
 
-3. Hand over to the courier. 
+3. Hand over to the courier.
 
+### [In Self-Managed](#tab/in-selfmanaged)
+
+Customers using Data Box Disk in Japan, Singapore, Korea and West Europe who have selected the self-managed shipping option to use their own shipping during order creation should follow the instructions below. 
```

Comment:
Change to: If you are using Data Box Disk in Japan, Singapore, Korea, and West Europe and have selected the self-managed shipping option during order creation, follow these instructions.